lib/oli/utils.ex

```diff
@@ -485,4 +485,14 @@ defmodule Oli.Utils do
       value -> value
     end
   end
+
+  @doc """
+  Returns the value passed in.
+
+  ## Examples
+  iex> Utils.identity(1)
+  1
+  """
+  @spec identity(any) :: any
+  def identity(x), do: x
```

Comment:
@gastonabella There is a `Function.identity` already in Elixir. See https://hexdocs.pm/elixir/main/Function.html#identity/1